src/julia-parser.scm

```diff
@@ -68,7 +88,9 @@
                              (pushprec (cdr L) (+ prec 1)))))
                      (pushprec (map eval prec-names) 1)
                      t))
-(define (operator-precedence op) (get prec-table op 0))
+(define (operator-precedence op) (get prec-table
+                                      (maybe-strip-op-suffix op)
```

Comment:
I tried benchmarking `parse(s)` for `s = string(:[$((:(a+b) for i=1:10000)...)])`, and removing the `maybe-strip-op-suffix` call from `operator-precedence` makes no detectable difference on my machine, so `operator-precedence` doesn't seem to be a problem.

However, there seems to be about an 8% slowdown overall in that benchmark compared to before this PR, so there must be something else in this PR that is the culprit.